oat-sa/tao-i18n-tools
# Feature/ms 130/extract strings

Ticket: https://oat-sa.atlassian.net/browse/MS-130

The goal of this PR is to create a script to extract all the strings from JS and Svelte files to a POT file.

### How does it work?

`npm run extract -- -s /path/to/source -d /path/to/dest/template.POT`

### How to run tests?

`npm run test`

Please check the `readme.md` in the PR for more detailed explanation.
